docs/application/native/guides/ui/dali/progressbar-n.md

```diff
@@ -0,0 +1,120 @@
+# ProgressBar
+
+The progress bar is a control to give the user an indication of the progress of an operation. The base class for the progress bar control is `Dali::Toolkit::ProgressBar`.
+
+In this tutorial, the following subjects are covered:
+
+[ProgressBar events](#1)<br>
+[Creating a ProgressBar](#2)<br>
+[ProgressBar Properties](#3)<br>
+[Customizing the ProgressBar Appearance](#4)<br>
+
+## Overview
+
+The ProgressBar has two modes to represent the progress. The default mode is Determinate mode.
```

Comment:
rewrite as The ProgressBar has two modes to represent the progress.

Reply:
Done. Deleted 'The default mode is Determinate mode.' sentence here.